CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
 If you are looking for the Ledger Releasing and Versioning Process then you can find it in
 [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
+
+## 9.0
+
+- Authorization of hot credentials for constitutional committee members is now only
+  possible for cold credentials that are present in the ledger state, either in the current
+  committee or in one of the proposals.
+- Voting is restricted to entities that are present in the ledger state
+- DRep votes will be removed whenever DRep unregisters
```

Comment:
I think using  the future tense "will be removed"  here is a little confusing - sounds like a plan rather than what is already implemented (especially after mentioning "now" in a previous sentence).   Better simplify to 
```suggestion
- DRep votes are removed whenever DRep unregisters
```
in my opinion.